### PR TITLE
Carousel object position

### PR DESCRIPTION
**Pull request by curtisadams28**
ADO card: ☑️ [AB#8213](https://dev.azure.com/S72/SHIFT72/_workitems/edit/8123)

## Description of work
Adds the object-position property to carousel images. This allows us to define how much each side is cropped.
`object-position: 0%` = Left aligned
`object-position: 50%` = Centred
`object-position: 70%` = Positioned 70% to the right
`object-position: 100%` = Right aligned
The image is still constrained by `object-fit: cover` so there will never be black bars.

## Checklist
- [x] CI tests are passing Github actions (inc. linting)
- [x] Key areas of the feature outlined for context and testing
- [x] If there are designs for this work are they noted here and in the ADO card 
- [x] Design review
- [x] Have checked this at multiple screen resolutions and range of browsers
- [x] Moved ADO card to Dev/done, checked link to Github, tagged "Review"
- [x] Updated changelog (if applicable)
- [x] I promise to document any new feature toggles/configurations in the appropriate documentation

### CODE REVIEW

**Review by l0ud0gg — APPROVED** [review]

**Comment by curtisadams28** [comment]
@PeculiarGoat Your window size needs to be smaller than the image.

**Review by brynyShift72 — APPROVED** [review]
LGTM

**Comment by PeculiarGoat** [comment]
@curtisadams28 I was shrinking and growing the window with no effect.

File: site/styles/_variables.scss [hunk 1]
```diff
@@ -33,6 +33,9 @@
   --carousel-height-sm: 560px;
   --carousel-height-lg: 620px;
   --carousel-bottom-gradient-height: 220px; // Increase this to 800px for light theme.
+  --carousel-image-object-position: 50%;
+  --carousel-image-object-position-sm: 50%;
+  --carousel-image-object-position-lg: 50%;
```
Comment (stajs) [inline site/styles/_variables.scss:38]: Like, this is fine and all, but perhaps using `center` would be better.

**Review by stajs — APPROVED** [review]

**Comment by PeculiarGoat** [comment]
@stajs I mean ticking/unticking the property in the browser inspector.

**Pull request merged**